### ISSUE TITLE
Trying to install logstash-kafka plugin onto container pre support for logstash 1.5 and the bin/plugin install mechanism

### ISSUE THREAD

**Issue by Analect** [pblittle/docker-logstash#67]
I realise that the this kafka plugin (https://github.com/joekiller/logstash-kafka) is part of the core logstash 1.5. However, I'm trying to get it installed to be part of this docker-logstash container running logstash version 1.4.2.
From the kafka plugin repo readme, I nsentered to the running logstash container and tried running from /opt/logstash: GEM_HOME=vendor/bundle/jruby/1.9 GEM_PATH= java -jar vendor/jar/jruby-complete-1.7.11.jar -S gem install logstash-kafka ... however, it didn't seem to like that, giving back an ENONET error "while executing gem .." no such file or directory, /home/myusername.
The only place it would have got this '/home/myusername' is that I supplied these with the -v option on starting the container, so I could have a local linked volume to feed the logstash.conf file.

Any suggestions how I might go about installing this plugin in a way that is appropriate to the existing installation in the container?
Thanks. Colum

**Comment by pblittle** [comment]
@Analect I'm not sure about this one without digging in. Have you gotten any further since leaving this comment?

**Comment by Analect** [comment]
@pblittle 
I ended up just installing 1.4.2 on my host machine ... having exhausted some of the other container solutions out there. Getting the kafka plugin working wasn't completely straight-forward, however, the following worked for me:
Downloading and untarring logstash_1.4.2.tar.gz
From https://logstash.jira.com/browse/LOGSTASH-1839, I ended up having to run:
"make vendor-jruby" 
then "bin/logstash deps" ... to install dependencies
and then was able to run that GEM_HOME etc... command (as per my initial comment) from the joekiller/logstash-kafka README.
This then appeared to give me the kafka input capability I was after.

My attempts to get this plugin working on verison 1.5.0.beta1 using the simplified bin/plugin install approach also failed, as per this issue: https://github.com/joekiller/logstash-kafka/issues/52

I'll close this out then. Thanks.
